framework/src/bluetooth/adaptation/bt-adatation-adapter.c

```diff
@@ -119,8 +137,33 @@ int bt_adapt_get_bonded_device_list(sq_queue_t *dev_list)
 }
 #endif
 
+static void __scan_start_cb(const bt_addr_le_t *addr, int8_t rssi, uint8_t adv_type, struct net_buf_simple *buf)
+{
+	bluetooth_event_param_t param = {0, };
+	bluetooth_le_device_info_t device_info = {0, };
+
+	BT_PRT("Enter\n");
+
+	param.event = BT_EVENT_LE_SCAN_RESULT_UPDATED;
+	param.result = 0;
+	param.param_data = &device_info;
+
+	__bt_le_event_proxy(BLUETOOTH_EVENT_REMOTE_LE_DEVICE_FOUND, &param);
+}
+
 int bt_adapt_start_le_scan(void)
 {
+	struct bt_le_scan_param param = {
+		.type = BT_HCI_LE_SCAN_PASSIVE,
+		.filter_dup = BT_HCI_LE_SCAN_FILTER_DUP_DISABLE,
+		.interval = 0x010,
+		.window = 0x010,
+	};
+
+	BT_DBG("Enter\n");
+
```

Comment:
In later, we should provide the error handling method such as "in progress" or "BT disable".
In this implemention, there is no return value for bt_le_scan_start. So never return "In progress" or "BT disable" error.
Please consider this in the later patchset.

Reply:
Thank you. I will consider to provide return values such as 'In progress' and 'BT disabled' and so on.